### DISCUSSION TITLE
Jak przenieść SQLIte na Azure SQL

### DISCUSSION THREAD

**Discussion by SensoryDeprivation** [AzureCommunityPL/group#121] [Pomoc]
Witam zebranych.
Mam pytanie (troche guglania nie przynioslo zadnego rozsadnego rezultatu).
Czy jest w miare prosty sposob, zeby z pliku sqlite .db ktory zawiera scheme, wrzucic to do Azure SQL.
I zeby nieco rozjasnic - serwis pythonowy, ktory ma sqlite baze wrzucam na Azure WebApp, i chce zeby korzystal z Azure SQL
Z gory dzieki za pomoc i sugestie

**Comment by wi5nia** [comment]
[To](https://docs.microsoft.com/en-us/azure/azure-sql/database/migrate-sqlite-db-to-azure-sql-serverless-offline-tutorial) nie wystarcza?

**Comment by kaluzaaa** [comment]
pierwszy link z google https://docs.microsoft.com/en-us/azure/azure-sql/database/migrate-sqlite-db-to-azure-sql-serverless-offline-tutorial

**Reply by wi5nia** [reply]
Nie zgapiaj

**Comment by SensoryDeprivation** [comment]
rzecz w tym, że mam DB scheme, i robienie całej tej akcji (szczegolnie jakbym to chcial zARM'owac albo oskryptowac) to jest jakas masakra.

Juz nawet pomijam to, ze mam tylko scheme, to sobie moge "na piechote" przerobic, ale jakbym miał tłustą baze, i musiał się tak nagimnastykowac to bym sie przecież ojsrał

Ale tak, ten link widziałem i zastanawiałem się nad rozsądniejszą opcją, bo tak to mi łatwiej baze przepisać z palca

**Reply by kaluzaaa** [reply]
zrobienie aż cli do postawienia bazy i potem wykonania na niej skryptu sql to grupa 6 komend wiec nie wiem o jakiej masakrze mówisz. klepnij skrypt sql z paląca i tyle.

**Reply by smereczynski** [reply]
A co jest niezwykłego w tej sytuacji? To normalne od dziesięcioleci, że migracja między silnikami to na ogół koszmar. Oracle do PGSQL czasem jest łatwiejsze, ale to tez nie zasada. Nie zautomatyzujesz tego. Face it. Odpuść sobie.

**Reply by SensoryDeprivation** [reply]
Dzięki za odpowiedź, obawiałem się tego trochę, ale jednak usłyszenie/przeczytanie uspokaja. 

Tak, wiem @kaluzaaa że to jest kilka komend, ale miałem małą nadzieję że to jest "portowalne".

**Reply by wi5nia** [reply]
Cuda się zdarzają, ale to tylko cloud

**Reply by smereczynski** [reply]
Pewnie że cuda się zdarzają, ale lepiej się miło zaskoczyć niż niemiło :) Sqlite to akurat przyjazna baza i rokuje na szanse automatu (semi automatu). Z innymi jest gorzej. Zawsze było i pewnie zawsze będzie. Tak działa ta branża.
